pipeline/07-export.R

```diff
@@ -396,6 +396,7 @@ assessment_pin_prepped <- assessment_pin_merged %>%
       ", ", loc_property_zip
     ),
     valuations_note = NA, # Empty notes field for Valuations to fill out
+    sale_ratio = NA # Initialize as NA so we can fill out with a formula later
```

Comment:
It's annoying that we can't do this calculation as part of the rest of the transformation that generates `assessment_pin_prepped`, but as with the `total_av` and `av_difference` columns, the problem is the result of using a formula instead of a static value: Since the formula needs to reference other cells by index, and we can't know the row number of any given value until after we start filtering for towns, we can't compute the formula at this stage of the pipeline.

We could get around this limitation by using a static value instead of a formula for the sale ratio, but I wanted to match the behavior of `total_av` and `av_difference` such that this column auto-updates if spreadsheet consumers decide to adjust the `Total` column.